docs/examples/model_ex/model_examples_pull_no_model_unc.py

```diff
@@ -2,7 +2,7 @@
 Pull plot, no model uncertainty
```

Comment:
Data vs Model, no model uncertainty

Reply:
Since we are already in the data vs model section, I prefer to keep it simpler.